modules/idle-scheduler/src/mock.rs

```diff
@@ -65,13 +66,16 @@ impl frame_system::Config for Runtime {
 
 parameter_types!(
 	pub const MinimumWeightRemainInBlock: Weight = 100_000_000_000;
+	pub const DisableBlockThreshold: BlockNumber = 6;
 );
 
 impl module_idle_scheduler::Config for Runtime {
 	type Event = Event;
 	type WeightInfo = ();
 	type Task = ScheduledTasks;
 	type MinimumWeightRemainInBlock = MinimumWeightRemainInBlock;
+	type RelayChainBlockNumberProvider = System;
```

Comment:
I would suggest to use a mock block number provider, to make it explicit in tests that the provider is from relay chain instead of system.